tools/segmetrics/segmetrics.xml

```diff
@@ -0,0 +1,96 @@
+        <param name="is_gt_unique"  type="boolean" checked="false" truevalue="-gt_unique"  falsevalue="" label="Ground truth is uniquely labeled" />
+
+        <section name="measures" title="Performance measures" >
+            <param name="dice" type="boolean" checked="true"  truevalue="-measure-dice" falsevalue="" label="Dice" />
+            <param name="seg"  type="boolean" checked="true"  truevalue="-measure-seg"  falsevalue="" label="SEG" />
+            <param name="jc"   type="boolean" checked="false" truevalue="-measure-jc"   falsevalue="" label="Jaccard coefficient" />
+            <param name="ji"   type="boolean" checked="true"  truevalue="-measure-ji"   falsevalue="" label="Jaccard index" />
+            <param name="ri"   type="boolean" checked="false" truevalue="-measure-ri"   falsevalue="" label="Rand index" />
+            <param name="ari"  type="boolean" checked="false" truevalue="-measure-ari"  falsevalue="" label="Adjusted Rand index" />
+            <param name="hsd_sym" type="boolean" checked="false" truevalue="-measure-hsd_sym" falsevalue="" label="Hausdorff distance (symmetric)" />
+            <param name="hsd_e2a" type="boolean" checked="false" truevalue="-measure-hsd_e2a" falsevalue="" label="Hausdorff distance (ground truth to segmented)" />
+            <param name="hsd_a2e" type="boolean" checked="false" truevalue="-measure-hsd_a2e" falsevalue="" label="Hausdorff distance (segmented to ground truth)" />
+            <param name="nsd"     type="boolean" checked="false" truevalue="-measure-nsd"     falsevalue="" label="Normalized sum of distances" />
+            <param name="o_hsd_sym" type="boolean" checked="true"  truevalue="-measure-o_hsd_sym" falsevalue="" label="Object-based Hausdorff distance (symmetric)" />
+            <param name="o_hsd_e2a" type="boolean" checked="false" truevalue="-measure-o_hsd_e2a" falsevalue="" label="Object-based Hausdorff distance (ground truth to segmented)" />
+            <param name="o_hsd_a2e" type="boolean" checked="false" truevalue="-measure-o_hsd_a2e" falsevalue="" label="Object-based Hausdorff distance (segmented to ground truth)" />
+            <param name="o_nsd"     type="boolean" checked="true"  truevalue="-measure-o_nsd"     falsevalue="" label="Object-based normalized sum of distances" />
+            <param name="fs" type="boolean" checked="true" truevalue="-measure-fs" falsevalue="" label="Falsely split objects per image" />
+            <param name="fm" type="boolean" checked="true" truevalue="-measure-fm" falsevalue="" label="Falsely merged objects per image" />
+            <param name="fp" type="boolean" checked="true" truevalue="-measure-fp" falsevalue="" label="Spurious objects per image" />
+            <param name="fn" type="boolean" checked="true" truevalue="-measure-fn" falsevalue="" label="Missing objects per image" />
+        </section>
+
+    </inputs>
+    <outputs>
+       <data format="csv" name="results" from_work_dir="results.csv" />

```

Comment:
in general we recommend to use tabular files - this is Galaxy can render tabular files currently better and more tools support tabular than csv

Reply:
@bgruening do you mean this here? https://usegalaxy.org/static/formatHelp.html#tab

> One or more columns of text data separated by tabs.

isn't this simply CSV with tabulator delimiters?

Reply:
Yes in Galaxy we call it `tabular` and its a simple tab separate file.

Reply:
ah ok, thanks for pointing this out, I will change the delimiter to `\t` then

Reply:
thanks a bunch!

Reply:
so I tested your suggestion and compared how the results are presented in Galaxy.

See for yourself, first, using `,` delimiters:
<img width="1871" alt="commas" src="https://user-images.githubusercontent.com/6557139/194726648-efaf40b6-64f3-42d5-aaca-07fbb217d3ca.png">

And then, using `\t` delimiters:
<img width="1880" alt="tabulators" src="https://user-images.githubusercontent.com/6557139/194726657-662c9073-2ad4-4435-abaa-a10f2af5a7a7.png">

As you can see, Galaxy fails to present the data in a proper tabular form when the columns are separated by tabs, but presents it nicely as a table when using commas. I guess I should stick to the commas then?

Reply:
the second dataset is not "tabular" as filetype. Thats why its not rendered correctly.
Its just a recommendation. In the end you need to check in which file format you want to pass your files around. All I can say is that traditionally more tools operate with tabular files - and as an overall community it would be better to stick to one (tabular vs csv) so that we do not need to interconvert that often.

Reply:
hm okay. Personally, I think that in case of doubt, standard formats (in this case CSV) should always be preferred to solution-specific formats. But I see that from a practical perspective, "tabular" files might be the better choice. By "filetype", do you mean the extension of the file (i.e. ".csv") or something else? Which should it be for "tabular" data?

Reply:
Do you think tsv (or tabular formatted csv), which we just call "tabular" is not a standard format? 

> By "filetype", do you mean the extension of the file (i.e. ".csv") or something else? Which should it be for "tabular" data?

Yes. Galaxy supports a bunch of datatypes (filetypes) https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/sample/datatypes_conf.xml.sample

And all those can have different metadata, can be sniffed etc ...

Reply:
All I'm saying is, in my personal impression, csv is all around the world, but this is the first time I'm hearing of tsv 🙂 Having switched the file type to `tsv`, the presentation in Galaxy looks as it should. Thanks for your help!